circle.yml

```diff
@@ -0,0 +1,27 @@
+machine:
+  node:
+    version: 7
+  pre:
+    - mkdir ~/.yarn-cache
+deployment:
+  staging:
+    branch: master
+    owner: artsy
+    commands:
+      - NODE_ENV=production make deploy env=staging
+  production:
+    branch: release
+    owner: artsy
+    commands:
+      - NODE_ENV=production make deploy env=production
```

Comment:
You'll want to make sure to update the deploy command to take `$CIRCLE_SHA1` into consideration [here](https://github.com/artsy/positron/blob/master/Makefile#L53). e.g. `git push --force git@heroku.com:positron-$(env).git $CIRCLE_SHA1:refs/heads/master`.